guides/tips-and-tricks.md

```diff
@@ -6,7 +6,13 @@ permalink: /guides/tips-and-tricks/
 
 # Tips and Tricks
 
+
+### Design Philosophy
+Although Dust is not completely logicless, so that you can get around some basic hurdles, it is still wise not to bloat templates with too much "business logic."  Business logic is essentially the way the data is stored.  If there's too much logic in a Dust template, it will be hard to read and hard to maintain.  Additionaly, you will eventually hit up against a wall with the amount of logic that Dust can handle, and will end up writing an endless number of helper functions to shoehorn your use case in.  This will just add to the difficulties in maintaining the template.  Let the server handle the business logic.
```

Comment:
Is this the right place for this paragraph? Also, it seems like we could shorten it to say "Don't put business logic in your template, even if it's possible."